src/testing/storage.zig

```diff
@@ -54,6 +54,8 @@ const log = std.log.scoped(.storage);
 // - likely that surrounding sectors also corrupt
 // - likely that stuff written at the same time is also corrupt even if written to a far away sector
 pub const Storage = struct {
+    pub const grid_blocks_max = stdx.div_ceil(constants.storage_size_max, constants.block_size);
```

Comment:
Should this use `storage_size_max - Zone.grid.start()` as the numerator? Also, why `div_ceil` instead of `@divFloor`?

Reply:
My original thinking was to make this some simple upper-bound -- the actual storage size is defined by the runtime `size` parameter to the `init` anyway, so being overly precise here didn't feel important. 

But, given that for convenience I need to round it to a free set shard size anyhow, I might as well compute this precisely. 

The high order bit here is that this now exists only in the testing code --- `superblock.grid_blocks_max` is no more, because the limit that matters now depends on the runtime value.